packages/vite/src/node/server/sourcemap.ts

```diff
@@ -18,3 +19,12 @@ export async function injectSourcesContent(
     })
   )
 }
+
+export function genSourceMapString(map: SourceMap | string | undefined) {
```

Comment:
**suggestion (dx, if-minor):** Please add a return type to export functions, because this will stabilize the code
So at first if someone changes the body of this function, all code-paths are checked against the return type. And second, if the return type needs to be changed by purpose, all calling parts are directly get checked and have better type-safty.

```suggestion
export function genSourceMapString(
  map: SourceMap | string | undefined
): string {
```

You can read more about it here: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md